styles/style.css

```diff
@@ -0,0 +1,66 @@
+html{
```

Comment:
You can do it this way, another is set image in the HTML and set it absolute while tinkering with the `z-index`